apis/managed-dimensions/hydra/index.ttl

```diff
@@ -49,6 +50,28 @@ md:ManagedDimensions
        ]
 .
 
+_:DELETE a hydra:Operation, schema:DeleteAction ;
+         hydra:method "DELETE" ;
+         code:implementedBy
+           [
+             a code:EcmaScript ;
+             code:link <file:handlers/resource#DELETE>
+           ] .
+
+schema:DefinedTermSet
+  a hydra:Class ;
+  query:cascadeDelete
+    [
+      sh:inversePath schema:inDefinedTermSet ;
+    ] ;
```

Comment:
This SHACL Path is converted into a SPARQL query path to find and delete terms contained by a term set